frontend/src/fixtures/reviewFormFixtures.js

```diff
@@ -0,0 +1,5 @@
+const ReviewFormFixtures = {
+  name: "Pasta",
+};
+
+export { ReviewFormFixtures };
```

Comment:
What is this for?  Fixtures are supposed to be stand in's for specific backend API endpoints?  What backend endpoint returns this?